monitoring/api/v3/api-client/list_resources_test.py

```diff
@@ -56,7 +56,7 @@ def test_list_metrics(client, capsys):
         client, PROJECT_RESOURCE, METRIC)
     stdout, _ = capsys.readouterr()
     regex = re.compile(
-        u'Delta CPU', re.I)
+        u'Delta vCPU', re.I)
```

Comment:
Nit: I think it's better to remove `vCPU` part (only check 'Delta')? But we can do it in another PR